topics.asciidoc

```diff
@@ -139,12 +141,14 @@ A summary of how the Lightning Network protocol works.
 === BOLT 1.1 / 2.0
 Future trends and developments on the Lightning Network.
 
+* channel factories
```

Comment:
Was already there as part of eltoo. It is pretty clear that they went fine with the penalty based asymmetrical protocol as the communication overhead would be too severe

Reply:
@renepickhardt  that was my bad. Probably was too late in the night for me when I made the changes :). I removed it.